build.gradle

```diff
@@ -40,17 +40,10 @@ jar {
 
 task copyDependencies(type: Copy) {
     doLast {}
-    from configurations.compileClasspath
```

Comment:
I believe this should be runtime>compile since this will be the minimal set of dependencies needed to run the SDK